src/StoreApi/Routes/Checkout.php

```diff
@@ -217,6 +217,7 @@ protected function get_route_post_response( WP_REST_Request $request ) {
 		 * Validate items etc are allowed in the order before the order is processed. This will fix violations and tell
 		 * the customer.
 		 */
+		$this->validate_terms_and_conditions( $request );
```

Comment:
I'm not sure about this change...

I don't think we should block API requests like this, especially if there is no 'schema' for it. I'd suggest removing this completely and only using client-side validation in the app.